core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala

```diff
@@ -231,11 +231,11 @@ class HadoopRDD[K, V](
       var reader: RecordReader[K, V] = null
       val inputFormat = getInputFormat(jobConf)
       HadoopRDD.addLocalConfiguration(new SimpleDateFormat("yyyyMMddHHmm").format(createTime),
-        context.stageId, theSplit.index, context.attemptNumber, jobConf)
+        context.stageId(), theSplit.index, context.attemptNumber(), jobConf)
```

Comment:
There are a number of trivial changes like adding parents to what are probably logically no-side-effect methods, and extra whitespaces